docs/units-and-global-variables.rst

```diff
@@ -1,13 +1,25 @@
+.. index:: literal;suffix
+.. _denominations:
+
 **************************************
 Units and Globally Available Variables
 **************************************
 
+Solidity defines several *denominations*, which can be applied as suffixes to
+:ref:`rational number literals<rational_literals>` to modify the value while still keeping it a literal.
+
+.. note::
+    Denominations should not be confused with :ref:`literal suffixes<literal_suffixes>`, which use
+    the same syntax, but can be defined by the user and result in a
+    :ref:`function call<function-calls-suffix-calls>`.
+
 .. index:: wei, finney, szabo, gwei, ether
```

Comment:
Now I'm wondering why exactly ``finney`` and ``szabo`` denominations got removed 😅

Reply:
AFAIK the only reason was that they were rarely used in practice.